bokehjs/src/lib/models/tools/actions/zoom_in_tool.ts

```diff
@@ -23,6 +23,11 @@ export class ZoomInTool extends ZoomBaseTool {
     this.register_alias("yzoom_in", () => new ZoomInTool({dimensions: "height"}))
   }
 
+  get_maintain_focus(): boolean {
```

Comment:
If you make this a `get` property then I think the code above can be simplified to
```
 this.plot_view.update_range(zoom_info, {scrolling: true, maintain_focus: this.model.maintain_focus})
```
and then no method at all would be needed in the zoom out tool

Reply:
I tried this, but the issue then is that ZoomOutTool redefines the maintain_focus property through:
```
this.define<ZoomOutTool.Props>(({Boolean}) => ({
      maintain_focus: [ Boolean, true ],
    }))
```
This then throws `attempted to redefine property`. I did manage to clean up the code a bit using your suggestion, but I had to name the base classes property differently (added a underscore prefix).